README.md

```diff
@@ -72,28 +72,20 @@ To run the tests:
 make test
 ```
 
-### Census Commands
-
-To translate the Census individual schema using current translations in Crowdin
-
-```
-pipenv run python -m eq_translations.cli.translate_census ../eq-survey-runner/data/en/census_individual_gb_eng.json out
-```
-
 ## Naming conventions
 
 ### Translation files
 
 Should be prefixed with the name of the schema to translate followed by `_translate_` followed by the [country code](https://en.wikipedia.org/wiki/ISO_3166-1) of the translations in a po format e.g.
 
 ```
-census_household_translate_cy.po
+household_translate_cy.po
 ```
 
 ## Managing translations
 
 When `gettext` is installed there are a number of command line utilities that can help with managing translations.
 
-To merge the translations from an already translated schema into another one, you can use `msgmerge`. For example `msgmerge census_household-cy.po census_individual.pot -o census_individual-cy.po` will merge matching Welsh translations from the Census household questionnaire into the Census individual questionnaire.
+To merge the translations from an already translated schema into another one, you can use `msgmerge`. For example `msgmerge household-cy.po individual.pot -o individual-cy.po` will merge matching Welsh translations from the household questionnaire into the individual questionnaire.
```

Comment:
Can update `household` and `individual` here too as possibly confusing outside the context of census?

Reply:
Added some more clear descriptions, thanks!